docs/development_suite/identity-and-access-management/user-management.md

```diff
@@ -0,0 +1,99 @@
+---
+id: user-management
+title: Manage Users
+sidebar_label: Manage Users
+---
+
+Mia-Platform Console enables certain Users to control other identities ability to access resources. This grants them ownership over the decision-making process regarding which resources are accessible and at what level of authorization.
+
+:::note
+To find out more about Roles check out the available [Capabilities](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#users-capabilities-inside-console) that can be assigned to an identity.
+:::
+
+In particular, a User can be assigned a Role on the following resources:
+
+* Company
+* Project
+* Runtime Environment
+
+Only Users with enough privileges will be granted access to the Identities administration portal and will be able to change identity Roles.
+
+:::caution Important: Git provider vs Console permissions
+The management of Roles and permissions aims at regulating access on resources solely at Console level.  
+Roles and permissions of Users on the same resources at the Git provider level are not managed in this Console area; thus, Roles at Git-level must be verified in order to prevent possible discrepancies that may appear between permissions the User has at the Console-level and those the User has at the Git repository level.
+
+For example, it could happen that a User who does not even have permissions to access a Company from the Console, is however in a Git group with an assigned Role that gives them Git-level permissions on these Company resources!
```

Comment:
```suggestion
For example, it could happen that a User who does not even have permissions to access a Company from the Console is, however, in a Git group with an assigned Role that gives them Git-level permissions on these Company resources!
```